.coveragerc

```diff
@@ -0,0 +1,47 @@
+# .coveragerc for edx-platform
+[run]
+data_file = reports/.coverage
+source =
+    cms
+    common/djangoapps
+    common/lib/calc
+    common/lib/capa
+    common/lib/xmodule
+    lms
+    openedx/core/djangoapps
+
+omit =
+    cms/envs/*
+    cms/manage.py
+    cms/djangoapps/*/migrations/*
+    cms/djangoapps/*/features/*
+    lms/envs/*
+    lms/djangoapps/*/migrations/*
+    lms/djangoapps/*/features/*
+    common/djangoapps/terrain/*
+    common/djangoapps/*/migrations/*
+    openedx/core/djangoapps/*/migrations/*
+
+branch = true
```

Comment:
Note: `branch = true` was originally only included `common/lib/calc/.coveragerc`.  I'm not sure if it is what we really want here, but I wanted to point it out and discuss.